Vivid-Project/frontend
# Feature/signup tests

## What is the change?
* Testing has been added for SignUp component
* Loading spinner has been added to the SignUp button 

## Is this a fix or a feature? 
* Feature

## Where should the reviewer start?
SignUp.test.js line 1

## How should this be tested?
From the signup page create a new user and see if a loading spinner appears instead of the singup button 
run all unit tests
run all integration tests 

## Relevant Screenshots:
![Screen Shot 2021-03-16 at 3 17 11 PM](https://user-images.githubusercontent.com/67513823/111380999-b833f400-866a-11eb-9724-2d8e837ac82b.png)
![Screen Shot 2021-03-16 at 3 18 23 PM](https://user-images.githubusercontent.com/67513823/111381105-ddc0fd80-866a-11eb-8276-630588f505ef.png)


## Any relevant tickets?
#58
